scrapyd_k8s/log_handler.py

```diff
@@ -0,0 +1,54 @@
+import os
+from kubernetes import client, watch
+import subprocess
+
+def get_last_n_lines(file_path, num_lines_to_check):
+    lines = []
+    try:
+        with open(file_path, 'r') as file:
+            lines = file.readlines()
+        return lines[-num_lines_to_check:]
```

Comment:
Performance-related tweak: seek to the end of the file, then read chunks in reverse order, and determine what the last lines are. This can be a significant improvement if the log files are huge. See e.g. [this SO](https://stackoverflow.com/a/23646049/2866660) or [`file_read_backwards`](https://pypi.org/project/file-read-backwards/).

Please keep this possible improvement for the end, let's first get the implementation done! (I needed a place to jot this down)